src/store/modules/app-gateway-svc/states.ts

```diff
@@ -0,0 +1,7 @@
+import { ModuleState } from './types';
+
+// eslint-disable-next-line import/prefer-default-export
+export const state: ModuleState = {
+  grpcClient: undefined as any,
+  httpHeader: undefined as any,
```

Comment:
`httpHeader` is more like `authHeader` at the moment. If we have to change it to `httpHeader` later on, then we can do it later on. I think it is more specific and easier to understand.